build_tools/cmake/iree_lit_test.cmake

```diff
@@ -52,10 +52,20 @@ function(iree_lit_test)
 
   get_filename_component(_TEST_FILE_PATH ${_RULE_TEST_FILE} ABSOLUTE)
 
+  # TODO(anyoneplease): use iree_add_data_dependencies to do this instead.
   list(TRANSFORM _RULE_DATA REPLACE "^::" "${_PACKAGE_NS}::")
   set(_DATA_DEP_PATHS)
+  set(_DATA_FILE_PATHS)
```

Comment:
DATA files don't work with this lit test stuff and I can't figure this out - I'd have thought what was below would work but there's so many stages of badness and it definitely won't work with installs (as it references the source dir).

Reply:
I also tried `if(TARGET ${_DATA_DEP})` but evaluation order changes that so it breaks random tests in other files.